manifests/agent/phpfpm.pp

```diff
@@ -0,0 +1,46 @@
+#
+# = Class: zabbix::agent::phpfpm
+#
+# This module installs Zabbix php-fpm sensor
+#
+class zabbix::agent::phpfpm (
+  $dir_zabbix_agentd_confd = $::zabbix::agent::dir_zabbix_agentd_confd,
+  $dir_zabbix_agent_libdir = $::zabbix::agent::dir_zabbix_agent_libdir,
+  $php_fpm_sock            = 'undef'
```

Comment:
If there is a distro package for Centos & Debian you should set default from these.